scripts/deploy.sh

```diff
@@ -18,6 +18,19 @@ npm run build
 cp package.json dist
 setup_git
 
+echo "Dry-run of semantic-version to determine whether the package will be published"
+# merge stderr and stdout and check for "no new version is released". If found, bail.
+# don't forget to capture the grep failure with `|| echo ""` so that the script doesn't fail
+# when the line isn't found
+npm run semantic-release -- -d 2>&1 | grep "no new version is released" > no_new_version.txt || echo "It looks like a new version will be released!"
```

Comment:
you don't need to dry run - I think that you can just validate the output of the existing command

Reply:
that's fair 👌